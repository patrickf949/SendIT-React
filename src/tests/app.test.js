import React from "react";
import renderer from "react-test-renderer";
import App from "../containers/app";

describe("App", () => {
  it("should render without crashing", () => {
    const wrapper = renderer.create(<App />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
