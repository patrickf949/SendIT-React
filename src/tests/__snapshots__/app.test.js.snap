// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without crashing 1`] = `
<div>
  <h1>
    Welcome to SendIT
  </h1>
</div>
`;
